core/src/main/java/tachyon/web/WebInterfaceConfigurationServlet.java

```diff
@@ -52,72 +64,19 @@ public WebInterfaceConfigurationServlet(MasterInfo masterInfo) {
-        + mTachyonConf.get(Constants.WEB_RESOURCES, ""));
-    request.setAttribute("tachyon.usezookeeper", ""
-        + mTachyonConf.getBoolean(Constants.USE_ZOOKEEPER, false));
-    request.setAttribute("tachyon.zookeeper.address", ""
-        + mTachyonConf.get(Constants.ZOOKEEPER_ADDRESS, ""));
-    request.setAttribute("tachyon.zookeeper.election.path", ""
-        + mTachyonConf.get(Constants.ZOOKEEPER_ELECTION_PATH, ""));
-    request.setAttribute("tachyon.zookeeper.leader.path", ""
-        + mTachyonConf.get(Constants.ZOOKEEPER_LEADER_PATH, ""));
-    request.setAttribute("tachyon.async.enabled", ""
-        + mTachyonConf.get(Constants.ASYNC_ENABLED, ""));
-    request.setAttribute("tachyon.max.columns", ""
-        + mTachyonConf.getInt(Constants.MAX_COLUMNS, 1000));
-    request.setAttribute("tachyon.max.table.metadata.byte", ""
-        + mTachyonConf.getBytes(Constants.MAX_TABLE_METADATA_BYTE, Constants.MB * 5));
-
-    // MasterConf
-    request.setAttribute("tachyon.master.journal.folder", ""
-        + mTachyonConf.get(Constants.MASTER_JOURNAL_FOLDER, ""));
-    request.setAttribute("FORMAT_FILE_PREFIX", ""
-        + mTachyonConf.get(Constants.MASTER_FORMAT_FILE_PREFIX, ""));
-    request.setAttribute("tachyon.master.hostname", ""
-        + mTachyonConf.get(Constants.MASTER_HOSTNAME, ""));
-    request.setAttribute("tachyon.master.port", ""
-        + mTachyonConf.getInt(Constants.MASTER_PORT, -1));
-    request.setAttribute("MASTER_ADDRESS", "" + mTachyonConf.get(Constants.MASTER_ADDRESS, ""));
-    request.setAttribute("tachyon.master.web.port", ""
-        + mTachyonConf.getInt(Constants.MASTER_WEB_PORT, 0));
-    request.setAttribute("tachyon.master.temporary.folder", ""
-        + mTachyonConf.get(Constants.MASTER_TEMPORARY_FOLDER, ""));
-    request.setAttribute("tachyon.underfs.hadoop.prefixes", "" 
-        + mTachyonConf.get(Constants.UNDERFS_HADOOP_PREFIXS, ""));
-    request.setAttribute("tachyon.test.mode", ""
-        + mTachyonConf.get(Constants.IN_TEST_MODE, "false"));
-    request.setAttribute("tachyon.master.retry", ""
-        + mTachyonConf.getInt(Constants.MASTER_RETRY_COUNT, 29));
-    request.setAttribute("tachyon.master.heartbeat.interval.ms", ""
-        + mTachyonConf.getInt(Constants.MASTER_HEARTBEAT_INTERVAL_MS, -1));
-    request.setAttribute("tachyon.master.min.worker.threads", ""
-        + mTachyonConf.getInt(Constants.MASTER_MIN_WORKER_THREADS, -1));
-    request.setAttribute("tachyon.master.max.worker.threads", ""
-        + mTachyonConf.getInt(Constants.MASTER_MAX_WORKER_THREADS, -1));
-    request.setAttribute("tachyon.master.worker.timeout.ms", ""
-        + mTachyonConf.getInt(Constants.MASTER_WORKER_TIMEOUT_MS, -1));
+    request.setAttribute("configuration", getSortedProperties());
 
     getServletContext().getRequestDispatcher("/configuration.jsp").forward(request, response);
   }
-}
 
+  private SortedSet<Pair<String, String>> getSortedProperties() {
```

Comment:
Maybe could cache it for few hours so no need to sort and iterate very time requests coming in.